packages/field/src/LionField.js

```diff
@@ -82,20 +95,18 @@ export class LionField extends FormControlMixin(
     return (this.inputElement && this.inputElement.value) || '';
   }
 
-  static get asyncObservers() {
-    return {
-      ...super.asyncObservers,
-      _setDisabledClass: ['disabled'],
-    };
+  constructor() {
+    super();
+    this.name = '';
+    this.submitted = false;
   }
 
   connectedCallback() {
+    this._delegateInitialValueAttr();
```

Comment:
@jorenbroekema could you add a comment here about why we want to add this before the connectedCallback? (I know it had something to do with testing the FormatMixin integrated with the LionField)?

Reply:
`// Initially delegating the value attribute should happen before it is evaluated by the FormatMixin.` maybe?
I'm not entirely sure what test failed but it had to do with this delegateInitialValueAttr triggering a second FormatMixin loop due to given `value attribute that is delegated` being different from the `formatted value` that is synced back to the user